source/read/distinct.txt

```diff
@@ -0,0 +1,152 @@
+For example, one document in the ``restaurant`` collection has a ``borough`` value of ``"Manhattan"``, and
+another has a ``borough`` value of ``"Queens"``. With the {+driver-short+}, you can
+retrieve all the distinct values that a field contains across multiple documents
+in a collection.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Distinct Command
+----------------
+
+To retrieve the distinct values for a specified field, call the ``mongoc_collection_read_command_with_opts()``
+function and direct it to use the ``distinct`` command. You must also specity the collection
+and field to retrieve data from.
+
+Retrieve Distinct Values Across a Collection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example retrieves the distinct values of the ``borough`` field in
+the ``restaurants`` collection:
+
+.. io-code-block::
+
+   .. input:: /includes/read/distinct.c
+      :start-after: start-distinct
+      :end-before: end-distinct
+      :language: c
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      { "values" : [ "Bronx", "Brooklyn", "Manhattan", "Missing", "Queens", "Staten Island" ], ... }
+
+The results show every distinct value that appears in the ``borough`` field
+across all documents in the collection. Although several documents have the same
+value in the ``borough`` field, each value appears in the results only once.
+
+Retrieve Distinct Values Across Specified Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can provide a **query filter** to the ``distinct`` command to find the distinct
+field values across a subset of documents in a collection. A query filter is an expression that specifies search
+criteria used to match documents in an operation.
+
+.. TODO
+.. For more information about
+.. creating a query filter, see :ref:`c-specify-query`.
+
+The following example retrieves the distinct values of the ``borough`` field for
+all documents that have a ``cuisine`` field value of ``"Italian"``:
+
+.. io-code-block::
+
+   .. input:: /includes/read/distinct.c
+      :start-after: start-distinct-query
+      :end-before: end-distinct-query
+      :language: c
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      { "values" : [ "Bronx", "Brooklyn", "Manhattan", "Queens", "Staten Island" ], ... }
+
+Modify Distinct Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``distinct`` command can be modified by passing options to the ``mongoc_collection_read_command_with_opts()``
+function. If you don't specify any options, the driver does not customize the operation.
+
+The following table describes some methods you can use to customize the
```

Comment:
S: I'm not sure if "methods" is the best way to describe these - maybe just "options":
```suggestion
The following table describes some options you can use to customize the
```